src/utils/TwoTokenHandshake/index.test.ts

```diff
@@ -0,0 +1,110 @@
+import { logout, startTTH } from ".";
+
+const performBEDHandshake = async (codeA) => "code_b";
+
+describe("Two Token Handshake", () => {
+  afterEach(() => {
+    //@ts-ignore
+    window.SPOTIM = {};
+  });
+
+  it("should call startTTH when its on window", async () => {
+    const windowStartTTHMock = jest.fn().mockReturnValue("bla");
+
+    //@ts-ignore
```

Comment:
Still need to be ignored?

Reply:
for some reason typescript does not use the global window type on .spec/test files.
I've tried to fix that but it took consumed too much time and with no success.
VFM. - I preferred to add this ignore line

Reply:
fixed global.d.ts, now removed